src/core/XMLprefs.cpp

```diff
@@ -549,6 +577,22 @@ int CXMLprefs::SetWithAttributes(const stringT &csBaseKeyName, const stringT &cs
   return iRetVal;
 }
 
+int CXMLprefs::SetWithAttributes(const stringT &csBaseKeyName, const stringT &csValueName,
```

Comment:
This is source code from one of the previous solutions and is no longer used. I've left the source code in case it might be useful in the future. I can also remove this if desired.